components/ui/icons/src/main/res/drawable/mozac_ic_edit_suggestion.xml

```diff
@@ -9,6 +9,6 @@
     android:viewportWidth="24.0"
     android:viewportHeight="24.0">
     <path
-        android:fillColor="@color/mozac_ui_icons_fill"
```

Comment:
Unfortunately, we want to use `@color/mozac_ui_icons_fill` for all our icons, so, that the consumer can easily override it. The `ui-icons` serves as a package for all our icons that can be used across Fenix and Focus so we don't want to ship hardcoded colours.

I am also looking at this as part of https://github.com/mozilla-mobile/fenix/issues/23350.

Reply:
Makes sense, will see if I can find anything else!